modal/sandbox.py

```diff
@@ -109,6 +111,12 @@ async def _load(self: _Sandbox, resolver: Resolver, _existing_object_id: Optiona
                 for path, volume in validated_volumes
             ]
 
+            open_port_proto = None
+            if open_ports:
+                open_port_proto = api_pb2.PortSpecs(
+                    ports=[api_pb2.PortSpec(port=port, unencrypted=unencrypted) for port, unencrypted in open_ports]
+                )
```

Comment:
Very minor but will the protofbuf default on the other end of the wire be equivalent to `api_pb2.PortSpecs(ports=[])`? If so  I think you could simplify to avoid needing the `None` initialization etc.

Reply:
The functionality will be the same, so I can do that.